typing/shape_reduce.ml

```diff
@@ -271,6 +271,22 @@ end) = struct
     | NComp_unit s -> Comp_unit s
     | NError s -> Error s
 
+  (* When interested only of in the uids of aliased modules we do not read_back
+    the entire shape of the module, just enough to unroll the chain of aliases.
+  *)
+  let read_back_aliases_uids env (nf : nf) =
```

Comment:
My feeling is that the code is correct, the comment is somewhat redundant (I think it is clear from the code), and the naming is slightly wrong, as you are not reading back (to generate a term). I would call this `reduce_aliases_for_uid` for example.